tests/bdd/features/file_storage.feature

```diff
@@ -10,13 +10,21 @@ Feature: Journals iteracting with the file system in a way that users can see
         And the journal directory should contain
             2013/07/23.txt
 
+    Scenario: Adding entries to a Folder journal with a custom extension should generate date files
+        Given we use the config "empty_folder_with_extension.yaml"
+        When we run "jrnl 23 July 2013: Testing folder journal."
+        Then we should get no error
+        And the journal directory should contain
+            2013/07/23.md
+
     Scenario: Adding multiple entries to a Folder journal should generate multiple date files
         Given we use the config "empty_folder.yaml"
         When we run "jrnl 23 July 2013: Testing folder journal."
         And we run "jrnl 3/7/2014: Second entry of journal."
         Then we should get no error
         And the journal directory should contain
             2013/07/23.txt
+            2014/07/23.txt
```

Comment:
Date doesn't line up with the test case in default US date layout -- either this should be changed to 2014/03/07.txt or the second `we run` line should be changed to: `And we run "jrnl 7/23/2014: Second entry of journal."`

Reply:
Fixed, I've updated the dates.